src/libraries/Common/src/System/Security/Cryptography/Asn1/OaepParamsAsn.xml.cs

```diff
@@ -11,11 +11,11 @@ namespace System.Security.Cryptography.Asn1
     [StructLayout(LayoutKind.Sequential)]
     internal partial struct OaepParamsAsn
     {
-        private static ReadOnlySpan<byte> DefaultHashFunc => new byte[] { 0x30, 0x09, 0x06, 0x05, 0x2B, 0x0E, 0x03, 0x02, 0x1A, 0x05, 0x00 };
+        private static ReadOnlySpan<byte> DefaultHashFunc => [0x30, 0x09, 0x06, 0x05, 0x2B, 0x0E, 0x03, 0x02, 0x1A, 0x05, 0x00];
```

Comment:
Styling question/preference....

With `new byte[] { x, y, z }` we almost always have the space between the brace and the first entry. Should we do the same here and consistently have `[ x, y, z ]`?

-- My own preference is yes, I find it immensely more readable to have it separated; but understand not everyone may agree

Reply:
Me too 🙂 

ideally there's a role that we can use to enforce whichever way we prefer..

Reply:
I talked with @CyrusNajmabadi on Discord and apparently no space is the Roslyn default (for various good reasons around matching tuples, list patterns, positional patterns, etc).

I still don't like it, but I'd guess its a losing battle 😄 
-- My brain likes to visually distinguish indices/parameters (`f(x, y)` or `f[x, y]`) from sets/collections (`{ x, y }` or `[ x, y ]`)

Reply:
Interestingly the [docs use the space too](https://github.com/dotnet/docs/blob/6ccfd5d56617f07c11966e0410e5686625b3e954/docs/csharp/language-reference/operators/snippets/shared/CollectionExpressionExamples.cs#L42) 😃.

Reply:
> for various good reasons around matching tuples, list patterns, positional patterns,

I'm curious what this means -- do you mean their formatter can't easily distinguish when a space would be needed and not, so it consistently doesn't use a space?

Reply:
We mean that this is consistent with tuples and list patterns. The existing features this is modeled after.